src/GraphQL/Execution/ExecutionResult.cs

```diff
@@ -11,6 +11,13 @@ namespace GraphQL
     /// </summary>
     public class ExecutionResult
     {
+        /// <summary>
+        /// Indicates if the operation included execution. If an error was encountered BEFORE execution begins,
+        /// the data entry SHOULD NOT be present in the result. If an error was encountered DURING the execution
+        /// that prevented a valid response, the data entry in the response SHOULD BE null.
+        /// </summary>
+        public bool Executed { get; set; }
```

Comment:
@Shane32 If there is a better way to distinguish between the errors before and after execution stage, then suggest.

Reply:
This seems fine to me.

Reply:
What about default value? Both `true`/`false` options have disadvantages.

Reply:
`false`